### ISSUE TITLE
dry run mode

### ISSUE THREAD

**Issue by badboy** [semantic-rs/semantic-rs#39]

**Comment by schultyy** [comment]
Should dry-run mode be the default?

**Comment by schultyy** [comment]
I would propose something like the `git clean` dry run. There it says:
```
would remove target/
would remove hello.txt
```

In our case this could look like:

```
[0] % cargo run -- -p ../test-project
     Running `target/debug/semantic-rs -p ../test-project`
semantic.rs 🚀
Analyzing your repository
Current version: 2.0.0
Analyzing commits
Commits analyzed. Bump would be Major
New version would be: 3.0.0
Would Changelog
Would create annotated git tag
```

**Comment by badboy** [comment]
Yup, dry-run by default sounds good. Maybe it makes sense to show the Changelog on the console in dry-run mode.

Also: semantic-release checks the env variable `CI` and if it is set it runs the real tool, otherwise a dry-run, we should do the same eventually.

**Comment by schultyy** [comment]
Agree!

**Comment by schultyy** [comment]
Resolved in #46
